demos/build_demos.sh

```diff
@@ -82,6 +88,15 @@ fi
 mkdir -p "$build_dir"
 
 (cd "$build_dir" && cmake -DCMAKE_BUILD_TYPE=Release "${extra_cmake_opts[@]}" "$DEMOS_PATH")
-cmake --build "$build_dir" -- "$NUM_THREADS"
+
+if [ -z "$build_targets" ]; then
+    cmake --build "$build_dir" -- "$NUM_THREADS"
+else
+    targets=($build_targets)
+    for t in "${targets[@]:1}"
+    do
```

Comment:
Use consistent style; `do` should be on the same line as `for`.

Reply:
Ok.